www/khur00/DU_3/task.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style_task.css">
+</head>
+
+<body>
+    <div>The font of this web page is not serif</div>
+    <div class="text_1">This text is red-ish using a named color</div>
```

Comment:
doporucuju psat nazvy s pomlckami, napr. text-1, text-2 - podle me je to prehlednejsi